_rules/SC1-2-3-video-alternative.md

```diff
@@ -0,0 +1,183 @@
+---
+name: video with audio has audio descriptions or transcript
+rule_type: composite
+description: |
+  This rule checks video elements with audio have an alternative for the video content as audio or as text.
+
+success_criterion:
+- 1.2.3 # Audio Description or Media Alternative (Prerecorded)
+
+atomic_rules:
+- SC1-2-video-audio-description
+- SC1-2-video-transcript
+- SC1-2-video-description-track
+- SC1-2-video-media-alternative
+
+authors:
+- Wilco Fiers
+- Brian Bors
+---
+
+## Test Procedure
+
+### Applicability
+
+The rule applies to any [non-streaming](#non-streaming) `video` element [visible](#visible), where the video contains audio.
```

Comment:
(question) applies to *any* or applies to *every* occurrence? AFAIK, "any" could be any subset of occurrences on a page.

Reply:
(editorial) grammar seems incorrect - "this rule applies to any visible non-streaming video element" or "this rule applies to any non-streaming video element that is visible" or such.

Reply:
Good point, changing this to "every".